packages/blade/src/components/Input/SelectInput/SelectInput.tsx

```diff
@@ -0,0 +1,79 @@
+import { BaseInput } from '../BaseInput';
+import type { BaseInputProps } from '../BaseInput';
+import type { IconComponent } from '~components/Icons';
+import { ChevronDownIcon, ChevronUpIcon } from '~components/Icons';
+import Box from '~components/Box';
+import { VisuallyHidden } from '~components/VisuallyHidden';
+import { getPlatformType } from '~utils';
+type SelectInputProps = Pick<
+  BaseInputProps,
+  | 'label'
+  | 'labelPosition'
+  | 'necessityIndicator'
+  | 'validationState'
+  | 'helpText'
+  | 'errorText'
+  | 'successText'
+  | 'name'
+  | 'isDisabled'
+  | 'isRequired'
+  | 'prefix'
+  | 'suffix'
+  | 'autoFocus'
+> & {
+  icon?: IconComponent;
+};
+
+const SelectInput = (props: SelectInputProps): JSX.Element => {
+  /**
+   * @TODO handle expanded state with Dropdown
+   */
+  const isPopupExpanded = false;
+  const value = '';
+
+  const { icon, ...baseInputProps } = props;
+
+  const platform = getPlatformType();
+
+  return (
+    <Box position="relative">
+      {/* @TODO Use this for form submissions */}
+      {platform !== 'react-native' ? (
+        <VisuallyHidden>
+          <input
+            tabIndex={-1}
+            required={props.isRequired}
```

Comment:
that native bubble looks weird since we set this 😢. probably we shall handle this with our validation state?

Reply:
I think we should ideally support native bubble. Consumers can always preventDefault during form submission (This sort of protects from scenario where someone forgets to write validations)

We're also supporting this in other inputs.